Assets/Scripts/UI/InfoWindow/SubsystemButtonManager.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Ships.Components;
+using UnityEngine;
+
+namespace UI.InfoWindow
+{
+    /// <summary>
+    ///     Used as a bitmask and type for IDamageable classes and Subsystem buttons.
+    /// </summary>
+    [Flags]
+    public enum Subsystem
+    {
+        None = 0,
+        Weapon = 1 << 0,
+        Engine = 1 << 1,
+        Hull = 1 << 2
+    }
```

Comment:
Not sure how well this will work in the long run, since we will likely want to have things like multiple targetable weapons on ships.

Reply:
This is mainly for hiding types of subsystems from being targetable. But yeah multiple weapons may be a problem down the line.